gravitee-node-cache/gravitee-node-cache-plugin-redis/src/main/java/io/gravitee/node/plugin/cache/redis/configuration/RedisConfigurationProvider.java

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.gravitee.node.plugin.cache.redis.configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.core.env.Environment;
+import org.springframework.util.StringUtils;
+
+/**
+ * @author Eric LELEU (eric.leleu at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public class RedisConfigurationProvider {
+
+    public static final String SENTINEL_PREFIX = ".sentinel";
+
+    public static RedisConfiguration from(Environment environment, String propertiesPrefix) {
+        RedisConfiguration config = new RedisConfiguration();
+
+        final var host = environment.getProperty(propertiesPrefix + ".host", String.class, "localhost");
+        final var port = environment.getProperty(propertiesPrefix + ".port", Integer.class, 6379);
+        final var password = environment.getProperty(propertiesPrefix + ".password", String.class);
+        final var useSsl = environment.getProperty(propertiesPrefix + ".useSsl", Boolean.class, false);
+
+        final var hostAndPort = HostAndPort.of(host, port).withPassword(password).withSsl(useSsl);
+        config.setHostAndPort(hostAndPort);
+
+        SentinelConfiguration sentinelConfiguration = new SentinelConfiguration();
+        sentinelConfiguration.setEnabled(isSentinelEnabled(environment, propertiesPrefix));
+        if (sentinelConfiguration.isEnabled()) {
+            sentinelConfiguration.setMaster(
+                environment.getProperty(propertiesPrefix + SENTINEL_PREFIX + ".master", String.class, "mymaster")
+            );
+            sentinelConfiguration.setMaster(environment.getProperty(propertiesPrefix + SENTINEL_PREFIX + ".password", String.class));
+
+            List<HostAndPort> sentinelNodes = getSentinelNodes(environment, propertiesPrefix);
+            sentinelNodes.forEach(node -> node.withPassword(hostAndPort.getPassword()).withSsl(hostAndPort.isUseSsl()));
+            sentinelConfiguration.setNodes(sentinelNodes);
+        }
+        config.setSentinelConfiguration(sentinelConfiguration);
+
+        if (useSsl) {
+            final var trustAll = environment.getProperty(propertiesPrefix + ".trustAll", Boolean.class, true);
```

Comment:
False by default.
APIM ratelimit redis set it to true, but let's not reproduce this unless you can bring justification